extension/package.json

```diff
@@ -21,7 +21,7 @@
     "@shared/api": "1.0.0",
     "@shared/constants": "1.0.0",
     "@shared/helpers": "1.0.0",
-    "@stellar/design-system": "^0.8.0",
+    "@stellar/design-system": "^0.8.1",
```

Comment:
had to make a very small adjustment to the SDS build chain because jsdom wasn't able to load the old version